ember/app/templates/components/tracking-page.hbs

```diff
@@ -13,7 +13,9 @@
       <div class="sidebar-pane" id="tab-overview">
         <h3>{{t "overview"}}</h3>
         <div class="sidebar-pane-content">
-          {{tracking-pilots-list pilots=pilots}}
+          {{#if fixCalc.fixes.[0].coordinate}}
```

Comment:
does `[0]` as a key path actually work?

why did you wrap this in a condition? I think the list should still show even if one of the coordinates is unavailable.

is there no other way to get to the last known coordinate (e.g. access `fixCalc` from within the `tracking-pilots-list` component)? in any case we should probably extract that logic into a `lastKnownPostion` computed property.